README.md

```diff
@@ -14,18 +14,95 @@
 
 </div>
 
-Vello is a 2d graphics rendering engine, using [`wgpu`].
+Vello is an experimental 2d graphics rendering engine written in Rust, using [`wgpu`].
 It efficiently draws large 2d scenes with interactive or near-interactive performance.
 
-<!-- Impressive picture here -->
-
-It is used as the rendering backend for [Xilem], a UI toolkit.
-
 Quickstart to run an example program:
 ```shell
 cargo run -p with_winit
 ```
 
+![image](https://github.com/linebender/vello/assets/8573618/cc2b742e-2135-4b70-8051-c49aeddb5d19)
+
+It is used as the rendering backend for [Xilem], a native Rust GUI toolkit.
```

Comment:
I don't really like the word "native" here - a subset of users will have the expectation of that meaning fully using the platform provided toolkits/widgets.

I've put a comment in [#linebender > rfcs#0001 - Masonry backend](https://xi.zulipchat.com/#narrow/stream/419691-linebender/topic/rfcs.230001.20-.20Masonry.20backend).